File: Bee 1015 - Distance Between Two Points/DistanceBetweenTwoPoints.py
#Bee 1015 - Distance Between Two Points
import math

val1 = map(float, input().split(" "))
x1, y1 = val1

val2 = map(float, input().split(" "))
x2, y2 = val2

dist = math.sqrt(((x2 - x1)*(x2 - x1)) + ((y2 - y1)*(y2 - y1)))
print('%.4lf' % dist)

File: Bee 1015 - Distance Between Two Points/DistanceBetweenTwoPoints.cpp
//Bee 1015 - Distance Between Two Points

#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main() {
    double x1, x2, y1, y2, res;

    cin >> x1 >> y1 >> x2 >> y2;
    res = sqrt(((x2 - x1)*(x2 - x1)) + ((y2 - y1)*(y2 - y1)));
    cout << fixed << setprecision(4) << res << endl;
    return 0;
}